pkg/services/sqlstore/migrations/app_settings.go

```diff
@@ -13,6 +13,7 @@ func addAppSettingsMigration(mg *Migrator) {
 			{Name: "enabled", Type: DB_Bool, Nullable: false},
 			{Name: "pinned", Type: DB_Bool, Nullable: false},
 			{Name: "json_data", Type: DB_Text, Nullable: true},
+			{Name: "secure_json_data", Type: DB_Text, Nullable: true},
```

Comment:
you need column migration for this, or a drop table migration / recreation, otherwise people who are running 3.0 pre release builds will not get the new column and get errors when updating.